### PR TITLE
Added documentations for CONTRIBUTING and COC.

### PR DESCRIPTION
**Pull request by krushiraj**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.next/
\ No newline at end of file
+.next/
+*.md
```
Comment (vinaypuppal) [inline .prettierignore]: Why do we need to ignore .md files?

File: Readme.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Coderplex Donation Page
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
```
Comment (vinaypuppal) [inline Readme.md]: This is actually already present in readme file

File: .prettierignore (hunk 1 above)
Comment (krushiraj) [reply .prettierignore]: Actually prettier is behaving very weird with .md files. It is changing almost half the content to some random character on save. So, I used different plugin to format .md files.

Comment (vinaypuppal) [reply .prettierignore]: ok okay 👍

**Comment by vinaypuppal** [comment]
In case you did not notice them

Screenshot 2019-08-25 at 3 11 19 AM

Please resolve these conflicts so we can merge this PR

**Comment by krushiraj** [comment]
I tried to see what is the content that is conflicting in the web editor itself. It shows the conflicts are too complex. I then thought the filename is messing up the things. I tried changing the file name but still it shows that there is conflict. I don't understand what is actually conflicting. Can you please help me out in letting me know what are the changes in my commit that are conflicting the with master.

**Comment by vinaypuppal** [comment]
@all-contributors please add @krushiraj for documentation

**Pull request merged**